web/src/typings/item.ts

```diff
@@ -4,4 +4,5 @@ export type ItemData = {
   usable: boolean;
   close: boolean;
   description?: string;
+  buttons?: any;
```

Comment:
😡

Reply:
I'm not sure how to do it differently.
I tried this, but it shows `Property 'length' does not exist on type '{ [key: number]: string; }'.` (_same with map_).
```ts
buttons?: {
  [key: number]: string;
};
```

Can't find any other solution 😕

Reply:
I think it's because map expects an array, not an object with kvp. I could be wrong though.

Reply:
Okay, that was easy 🤦🏾‍♂️ Learned something new.